cmd/root.go

```
@@ -0,0 +1,62 @@
+                        logger.Error("Command execution failed", zap.Error(err))
+                } else {
+                        fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+                }
+                os.Exit(1)
+        }
+}
+
+func init() {
+        rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "Enable verbose logging")
+        rootCmd.Flags().StringVar(&repoURL, "repo-url", "", "GitHub repository URL (required)")
+        rootCmd.MarkFlagRequired("repo-url")
```

golangci-lint (1.64.8)

Error return value of `rootCmd.MarkFlagRequired` is not checked

(errcheck)